.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: Release
+
+# Adapted from https://pdm-project.org/latest/usage/publish/
+on:
+  release:
+    types: [published]
+
+jobs:
+  pypi-publish:
+    name: Publish to PyPI
+    environment: release
+    runs-on: ubuntu-latest
+    permissions:
+      # This permission is needed for private repositories.
```

Comment:
Is this comment related to what we are doing?

Reply:
This comment is from the original code snippet that I copied. For public repos, I assume that this permission is not needed, but it doesn't hurt to have it, and we have no guarantee that it works without it. I also prefer to keep the comment as an indication that it might not be necessary for us.